### DISCUSSION TITLE
Generate report

### DISCUSSION THREAD

**Discussion by AayushiRV** [The-OpenROAD-Project/OpenROAD#3771] [General]
Is there any way to generate a report for max cap, max fanout and max slew after “repair_design” is applied?

**Comment by maliberty** [comment]
If you are using ORFS then we already do that in 3_4_place_resized.log.  If any flow you can insert the relevant commands.

**Comment by AayushiRV** [comment]
I am not using ORFS.

**Comment by maliberty** [comment]
You can add ``` report_check_types -max_slew -max_capacitance -max_fanout``` to whatever you are using then

**Comment by AayushiRV** [comment]
OK, I am using this, if after using this command nothing is shown does that means all the violations are cleared?

**Reply by vijayank88** [reply]
can you share the scripts for reference to see whether it placed on correct order? @AayushiRV

**Reply by AayushiRV** [reply]
I just checked, I was actually using the tag -violators at the end, so it was not showing me any report as the violations were met. Now with the above command I am getting the report on my terminal. Thanks for help.
